src/lib/es2017.sharedmemory.d.ts

```diff
@@ -11,13 +11,13 @@ interface SharedArrayBuffer {
      * Returns a section of an SharedArrayBuffer.
      */
     slice(begin: number, end?: number): SharedArrayBuffer;
-    readonly [Symbol.species]: SharedArrayBuffer;
     readonly [Symbol.toStringTag]: "SharedArrayBuffer";
 }
 
 interface SharedArrayBufferConstructor {
     readonly prototype: SharedArrayBuffer;
     new (byteLength: number): SharedArrayBuffer;
+    readonly [Symbol.species]: SharedArrayBuffer;

```

Comment:
Shouldn't this also be `SharedArrayBufferConstructor`?

Reply:
@DanielRosenwasser Oh... Thank you!